xdsl/dialects/vector.py

```diff
@@ -62,7 +63,9 @@ class LoadOp(IRDLOperation):
     base = operand_def(MemRefType)
     indices = var_operand_def(IndexType)
     result = result_def(VectorType)
+    nontemporal = opt_prop_def(BoolAttr)
```

Comment:
```suggestion
    nontemporal = opt_prop_def(BoolAttr, default_value=BoolAttr.from_bool(False))
```
These properties have defaults in MLIR.